apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -213,6 +213,22 @@ type OpenTelemetryCollectorSpec struct {
 	// +optional
 	InitContainers []v1.Container `json:"initContainers,omitempty"`
 
+	// AdditionalContainers allows injecting additional containers into the Collector's pod definition.
+	// These sidecar containers can be used for authentication proxies, logging shipping sidecars, agents for shipping
```

Comment:
perhaps `s/logging shipping/log shipping` I am not a native speaker so not 100 sure :)